x/multisig/types/keygen.go

```diff
@@ -157,6 +157,23 @@ func (m Key) GetMinPassingWeight() sdk.Uint {
 	return m.Snapshot.CalculateMinPassingWeight(m.SigningThreshold)
 }
 
+// GetPubKey returns the public key of the given participant
+func (m Key) GetPubKey(p sdk.ValAddress) (exported.PublicKey, bool) {
+	pubKey, ok := m.PubKeys[p.String()]
+
+	return pubKey, ok
```

Comment:
```suggestion
	return m.PubKeys[p.String()]
```
I didn't check the compiler if this works though

Reply:
No it wouldn't